app/ide-desktop/lib/dashboard/src/authentication/src/config.ts

```diff
@@ -17,7 +17,7 @@ const CLOUD_REDIRECTS = {
      * when it is created. In the native app, the port is unpredictable, but this is not a problem
      * because the native app does not use port-based redirects, but deep links. */
     development: newtype.asNewtype<auth.OAuthRedirect>('http://localhost:8081'),
-    production: newtype.asNewtype<auth.OAuthRedirect>('https://cloud.enso.org'),
+    production: newtype.asNewtype<auth.OAuthRedirect>('http://localhost:8080'),
```

Comment:
add here fixme (and create issue) that this is temporary and should be fixed to support both gui watch and cloud environment